app/resource/views.py

```diff
@@ -0,0 +1,8 @@
+from flask import render_template
+from flask.ext.login import login_required
+from . import resource
+
+@resource.route('/')
+@login_required
+def view():
```

Comment:
Let's call the function for the default route `index`. The corresponding template should be called `resource/index.html`.